src/kili/services/copy_project/__init__.py

```diff
@@ -0,0 +1,139 @@
+"""
+Copy project implementation.
+"""
+from typing import Dict, Optional
+
+from kili.authentication import KiliAuth
+from kili.mutations.project import MutationsProject
+from kili.queries.project import QueriesProject
+from kili.queries.project_user import QueriesProjectUser
+
+
+class CopyProject:  # pylint: disable=too-few-public-methods
+    """
+    Class for copying an existing project.
+    """
+
+    FIELDS_PROJECT = ["title", "jsonInterface", "inputType", "description", "id"]
```

Comment:
for some clients, the json_interface can be very big. it does not look like the template ones that we have when creating projects. Only query the json_interface if copy_json_interface is set to true. Same way that you did with copy_quality_settings